sources/editor/Stride.GameStudio.Tests/XunitAttributes.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
I would rather not have that for now. It might or might not be needed, but let's isolate fixes. Otherwise, we can't determine what fixed what.

Reply:
How about using the [CollectionAttribute](https://xunit.net/docs/running-tests-in-parallel.html) on `TestCopyPasteProperties` and `TestCopyPasteWithEntities` to mark those three tests which must not run in parallel?

Reply:
If that helps passing the tests without changing too much the delay, that might be a good alternative indeed.